### PR TITLE
datadog crate

### PR DESCRIPTION
**Pull request by Shella**

### CODE REVIEW

File: datadog/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+{
+    ddtags
+        .iter()
+        .map(|(k, v)| [k.clone(), v.clone()].join(":"))
+        .collect::<Vec<_>>()
+        .join(",")
+        .serialize(serializer)
+}
+
+pub async fn send_event<T>(value: &T, dd_api_key: String) -> Result<(), Error>
```
Comment (zmanian) [inline datadog/src/lib.rs:35]: Is there a benefit for `send_event` being generic here?

It seems like the method is really only intended to be passed in Event structs.

Comment (Shella) [reply datadog/src/lib.rs:35]: this is a months old WIP. not ready for review, let you know after connecting to mina.

Comment (zmanian) [reply datadog/src/lib.rs:35]: Sorry. I was just taking getting a sense of what was going on.

Comment (Shella) [reply datadog/src/lib.rs]: Made it use the Event struct.

**Comment by Shella** [comment]
Opening up for initial review since this diff is so old and basic functionality is there. We can chat more about future work & `delphi` integration in our meeting tomorrow!

**Review by zmanian — APPROVED** [review]

**Pull request merged**